ecs/execution-role.tf

```diff
@@ -160,10 +167,18 @@ resource "aws_iam_role_policy" "ecs-task-execution-secrets-policy" {
           "kms:Decrypt",
           "kms:GenerateDataKey" # for writing to an encrypted S3 bucket
         ],
-        "Resource" : flatten([
-          var.secrets_arns,
-          var.kms_key_arns,
-        ]),
+        "Resource" : local.kms_and_secret_arns,
+      }
+    ]
+    }) : jsonencode({
+    "Version" : "2012-10-17",
+    "Statement" : [
+      {
+        "Effect" : "Deny",
+        "Action" : [
+          "secretsmanager:GetSecretValue",
+        ],
+        "Resource" : "arn:aws:secretsmanager:${var.region}:*:secret:can-t-be-blank",
```

Comment:
How about just skip the creation of this role policy if kms and secret arns is missing?